Dockerfile

```diff
@@ -1,8 +1,9 @@
-FROM python:3.7-alpine
+# gcloud doesn't support 3.7
+FROM python:3.6-alpine
```

Comment:
This is bad, Google probably won't be adding support for newer versions anytime soon. We discussed it IRL and the best option, both for code and maintainability seems to be to use the APIs directly. There is a [python wrapper](https://github.com/googleapis/google-cloud-python) that already supports this, so we'll have a look at that.

If it's not enough, we'll just use `pyenv` to have multiple python versions, so we can update the main app and still use the `gcloud` client